### PR TITLE
overlap for src-dest with the same center in drawio

### PR DESCRIPTION
**Pull request by haim-kermany**
handling the case of one square with two self loop.
in general, when there is a self loop, drawio adds point, and the line looks OK
however, when there are two self loops, drawio draw them one over the other.

### CODE REVIEW

**Comment by zivnevo** [comment]
@haim-kermany , can u please explain how such a case can happy in real life? If I understand correctly, the connectivity from a square to itself will always be represented by a single connection set.

**Comment by haim-kermany** [comment]
> @haim-kermany , can u please explain how such a case can happy in real life? If I understand correctly, the connectivity from a square to itself will always be represented by a single connection set.

That is a good question.
I should probably asked it on Thursday morning

File: pkg/drawio/layout.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ import (
 
 const (
 	minSize         = 10
+	twoMinSize      = 20
+	threeMinSize    = 20
```
Comment (zivnevo) [inline pkg/drawio/layout.go:30]: `twoMinSize` and `threeMinSize` are not being used.

File: pkg/drawio/layout.go [hunk 2]
```diff
@@ -26,6 +26,9 @@ import (
...
+	fourMinSize     = 40
```
Comment (zivnevo) [inline pkg/drawio/layout.go:31]: suggest to rename to `fourTimesMinSize`

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/drawio/layout.go (hunk 1 above)
Comment (haim-kermany) [reply pkg/drawio/layout.go]: fixed

File: pkg/drawio/layout.go (hunk 2 above)
Comment (haim-kermany) [reply pkg/drawio/layout.go]: fixed

**Review by zivnevo — APPROVED** [review]

**Pull request merged**